test/Microsoft.Extensions.Localization.Tests/ResourceManagerStringLocalizerTest.cs

```diff
@@ -59,10 +59,35 @@ public void EnumeratorCacheIsScopedByAssembly()
             Assert.Equal(expectedCallCount, resourceAssembly2.GetManifestResourceStreamCallCount);
         }
 
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void ResourceManagerStringLocalizer_GetAllStrings_MissingResourceThrows(bool includeParentCultures)
+        {
+            // Arrange
+            var resourceNamesCache = new ResourceNamesCache();
+            var baseName = "testington";
+            var resourceAssembly = new TestAssemblyWrapper("Assembly1");
+            var resourceManager = new TestResourceManager(baseName, resourceAssembly.Assembly);
+            var localizer = new ResourceManagerWithCultureStringLocalizer(
+                resourceManager,
+                resourceAssembly.Assembly,
+                baseName,
+                resourceNamesCache,
+                CultureInfo.CurrentCulture);
+
+            // Act & Assert
+            Assert.Throws<MissingManifestResourceException>(() =>
```

Comment:
`Assert.Equal` should perfom case sensitive comparison. You don't need to specify the comparer.